webapp/components/admin_console/storage_settings.jsx

```diff
@@ -202,6 +204,25 @@ export default class StorageSettings extends AdminSettings {
                     disabled={this.state.driverName !== DRIVER_S3}
                 />
                 <BooleanSetting
+                    id='amazonS3SignV2'
+                    label={
+                        <FormattedMessage
+                            id='admin.image.amazonS3SignV2Title'
```

Comment:
These strings need to be added to `webapp/i18n/en.json` so that the translators will know they need to be localized.

Also, can you review the text itself, @jasonblais? Here's Amazon's documentation on using Signature Version 2 vs Version 4: http://docs.aws.amazon.com/general/latest/gr/signature-version-2.html